src/planner.cpp

```diff
@@ -65,28 +64,31 @@ void Planner::setMap(const nav_msgs::OccupancyGrid::Ptr map) {
     }
   }
 
-  voronoiDiagram.initializeMap(width, height, binMap);
-  voronoiDiagram.update();
-  voronoiDiagram.visualize();
+  // voronoiDiagram.initializeMap(width, height, binMap);
+  // voronoiDiagram.update();
+  // voronoiDiagram.visualize();
 //  ros::Time t1 = ros::Time::now();
 //  ros::Duration d(t1 - t0);
 //  std::cout << "created Voronoi Diagram in ms: " << d * 1000 << std::endl;
 
   // plan if the switch is not set to manual and a transform is available
-  if (!Constants::manual && listener.canTransform("/map", ros::Time(0), "/base_link", ros::Time(0), "/map", nullptr)) {
+  if (!Constants::manual && listener.canTransform("/map", ros::Time(0), "/odom", ros::Time(0), "/map", nullptr)) {
```

Comment:
It will be great if you can provide rosparam for `map` and `odom` frame ids.

Reply:
Actually, I do plan to move all the constants and params into a separate yaml. I had to revert it halfway because of some issues.
Will do it separately after I fix the branches properly.
I will create branches both in path-planner and wheeltec-robot after the current merges.